CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to this repository
+
+## Getting started
+
+There are two different "modes" for developing this plugin. You can either work on just the plugin, pointed at Grafana cloud (easier to set up), or if you need to work across the entire stack you'll need to set up all the dependent components.
+
+### Dependencies
```

Comment:
we need SQLite as well, for grafana database.

Reply:
I think that comes with the docker image